### PR TITLE
#164796883 Integrate code coverage using Coveralls

### PR DESCRIPTION
**Pull request by micah-akpan**
#### Description
Integrate code coverage using Coveralls to ensure all tests in project cover most use and edge cases
#### Type of change
New feature (Non-breaking change which adds functionality)
#### How Has This Been Tested?
It has not been tested yet, because implemented Travis Integration is not on the configured base branch
#### Checklist:
- My changes generate no new warnings
#### PT
[164796883](https://www.pivotaltracker.com/story/show/164796883)

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: a5YJZKNBov5cQcXHYLPYhm4CvoFUyN0D2
```
Comment (Temmyogunbo) [inline .coveralls.yml]: Please add a new line

Comment (micah-akpan) [reply .coveralls.yml]: > Please add a new line

Okay. Thanks for pointing that out
